src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -173,7 +173,10 @@ export default function InstallModal({ appName, backdropClick }: Props) {
   return (
     <span className="modalContainer">
       {gameInfo?.game?.title ? (
-        <div className="modal">
+        <div className="modal close-icon-container">
```

Comment:
I don't think we need to add the `close-icon-container` class here, you can add `position: relative` to `.modal` in src/screens/Library/components/InstallModal/index.css

I think you can also move the `.close-icon` css inside that InstallModal/index.css file instead of the App.css, since this close button is only used in the InstallModal

Reply:
Got it, thanks for your feedback! it's on my last commit